src/pyfaf/storage/report.py

```diff
@@ -70,7 +70,10 @@ def bugs(self):
 
     @property
     def oops(self):
-        return self.get_lob('oops').decode('utf-8')
+        try:
+            return self.get_lob('oops').decode('utf-8')
```

Comment:
Why not just check return value of `get_lob` and only `decode` it if it is not `None`?